web-frontend/src/api/account.tsx

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+
+const api = axios.create({
+  baseURL: import.meta.env.VITE_API_URL,
+});
+export const getAllAccounts = () => api.get('accounts').then((res) => res.data);
```

Comment:
Can't we use the **useResource** standard we discussed to reach the endpoints!? 🤔